internal/kafka/internal/services/kafka_tls_certificate_management/kafka_tls_certificate_management_config.go

```diff
@@ -0,0 +1,131 @@
+package kafka_tls_certificate_management
```

Comment:
Can we put this into the config package?
This package is inside services which I'd expect belong to the domain logic

Reply:
It can be moved there for consistency reason but I find that the structure we've is a bit technical - we could revisit this structure to be more functional oriented.

Reply:
I am not sure I understand what do you mean with "a bit technical". The reasoning to put in in the other package is due to it belongs to a different layer than the domain logic

Reply:
Sorry I should have been clear.
We've a technical oriented layer approach where we put together modules that are technically similar: configs go into a config package, endpoints handlers goes onto the handler packages etc
 
Another approach is a functional approach where all modules for a particular function/feature are together. For example:
```
kafka_create_use_case/
  kafka_create_endpoint.go
  kafka_create_service.go
  kafka_create_config.go
  kafka_create._foo_bar_for_create_functionilty.go
  ....etc.go
```